File: math/sum_naturals.c
#include <stdio.h>

int sum_naturals(int n) { return n * (n + 1) / 2; }

int main() {
  int n = 5;
  printf("Sum of naturals until %d is %d\n", n, sum_naturals(n));
  return 0;
}

File: math/sum_naturals.go
package main

import "fmt"

func sum_of_naturals(n int) int {
	return n * (n + 1) / 2
}

func main() {
	n := 5
	fmt.Printf("Sum of naturais until %d id %d\n", n, sum_of_naturals(n))
}
